.github/workflows/release.yml

```diff
@@ -54,19 +79,21 @@ jobs:
           cp connector/target/*.jar staging
           cp connector/.flattened-pom.xml staging/kusto-spark_$sparkversion_$scalaversion-$version.pom
       - name: Github Release
-        uses: anton-yurchenko/git-release@v5.0.1
+        uses: docker://antonyurchenko/git-release:v6
```

Comment:
Why use an external release plugin

https://docs.github.com/en/actions/publishing-packages/publishing-java-packages-with-maven#publishing-packages-to-github-packages can this be used ?